TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/project.json

```diff
@@ -10,9 +10,9 @@
     ]
   },
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24027"
+    "NETStandard.Library": "1.5.0-rc3-24113-00"
   },
   "frameworks": {
-    "netstandard1.5": {}
+    "netstandard1.6": {}
```

Comment:
Libraries shouldn't need a runtime in their project.json since those aren't directly runnable.